src/components/Search/SearchListWithHeader.tsx

```diff
@@ -44,7 +45,7 @@ function mapToItemWithSelectionInfo(item: TransactionListItemType | ReportListIt
 }
 
 function SearchListWithHeader(
-    {ListItem, onSelectRow, status, hash, data, searchType, isMobileSelectionModeActive, setIsMobileSelectionModeActive, ...props}: SearchListWithHeaderProps,
+    {ListItem, onSelectRow, queryJSON, hash, data, searchType, isMobileSelectionModeActive, setIsMobileSelectionModeActive, isSearchResultsMode = false, ...props}: SearchListWithHeaderProps,
```

Comment:
ok hear me out 😅 controversial suggestion:
rename
```
isSearchResultsMode -> isCustomQueryMode`
```

Usually Im happy with names specific to what they describe and "mode" of SearchResults sounds like a good name.

However Search is becoming more and more complex. And in here we're introducing 2 new concepts: "SerchResultsMode" and "isCustomQuery".
So maybe everything will be simpler if we just drop the results naming, and use naming related to `customQuery` everywhere?